Lesson2/src/Main.java

```diff
@@ -1,5 +1,68 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+        byte b2 = 0b1000;
+        byte b8 = 010 ;
+        byte b16 = 0xC ;
+        byte b10 = 2;
+        int sumByte = b2+b8+b16+b10;
+
+        short s2 = 0b11;
+        short s8 = 010;
+        short s16 = 0xD;
+        short s10 = 6;
+        int sumShort = s2+s8+s16+s10;
```

Comment:
таке саме ну і там нище
```suggestion
        int sumShort = s2 + s8 + s16 + s10;
```